<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MFBtnDlg</class>
 <widget class="QDialog" name="MFBtnDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>427</x>
     <y>480</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_preview">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>152</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string>预览</string>
   </property>
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>18</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>591</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>基本设置</string>
    </attribute>
    <widget class="QGroupBox" name="group_macro">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>使用脚本</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QComboBox" name="combo_macroIndex">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>18</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>36</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>脚本</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_new">
      <property name="geometry">
       <rect>
        <x>172</x>
        <y>17</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>写脚本</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>261</width>
       <height>54</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_bit">
        <property name="text">
         <string>添加位操作</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_pic">
        <property name="text">
         <string>添加画面切换</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_word">
        <property name="text">
         <string>添加字操作</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_del">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>241</width>
       <height>281</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>功能列表：</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>291</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>字地址设置</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>字地址： </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_operatingAddr_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LW0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_wordAddr">
      <property name="geometry">
       <rect>
        <x>232</x>
        <y>19</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>数据类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dataType">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>16位整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16位正整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位正整数</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16位BCD码</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位BCD码</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>32位浮点数</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>操作类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_operateType_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>写入数据</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>数值加</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>数值减</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>连续加</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>连续减</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>138</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>常数：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_value">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>137</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_const">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>137</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LW0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_wordAddr_const">
      <property name="geometry">
       <rect>
        <x>229</x>
        <y>135</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_const">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>动态控制</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>319</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>画面切换设置</string>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>24</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>操作类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_operType">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>22</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>跳转到下一个画面</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>跳转到指定画面</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>打开窗口</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关闭窗口</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>返回上一个画面</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>56</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>画面名称：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_ScreenIndex">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>54</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>84</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>跳转时注销用户</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>291</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>位地址设置</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>位地址 ：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edit_operatingAddr">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>133</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>LB0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_bitAddr">
      <property name="geometry">
       <rect>
        <x>234</x>
        <y>19</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>操作类型：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_operateType">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>置位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>复位</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>交替</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>点动</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>执行方式：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_executeType">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>按下为1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>按下为0</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_btntype">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>控件类型</string>
   </property>
   <widget class="QLabel" name="label_Type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 80, 0);</string>
    </property>
    <property name="text">
     <string> 多功能按钮</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_tips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>152</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>提示</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 63, 255);</string>
    </property>
    <property name="text">
     <string>支持三种操作：

字操作

位操作

画面操作</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
